src/pokedex_screen.c

```diff
@@ -2117,3 +2117,9 @@ bool8 sub_81052D0(u8 a0)
     }
     return FALSE;
 }
+
+void sub_8106B34(void){
+    if(gMain.newKeys & 0x8){
```

Comment:
Style guide: space between `if/for/while/do` keywords and opening parenthesis
Repository knowledge: `gMain.newKeys & 0x8` --> `JOY_NEW(START_BUTTON)`

Reply:
Fixed